packages/apputils/src/dialog.ts

```diff
@@ -60,7 +60,7 @@ export function showErrorMessage(
       title: title,
       body: body,
       buttons: buttons
-    }).then(() => {
+    }).finally(() => {
```

Comment:
Looking at this with @jasongrout - it isn't clear that TypeScript is having trouble with the typing of `finally` and it is a recent addition. Maybe use `catch`?

Reply:
ping @vidartf